aea/aea.py

```diff
@@ -87,16 +92,24 @@ def __init__(
 
         self.max_reactions = max_reactions
         self._task_manager = TaskManager()
-        self._decision_maker = DecisionMaker(identity, wallet, ledger_apis)
+        decision_maker_handler_class = (
+            DefaultDecisionMakerHandler
+            if decision_maker_handler_class is None
+            else decision_maker_handler_class
+        )
+        decision_maker_handler = decision_maker_handler_class(
+            identity=identity, wallet=wallet, ledger_apis=ledger_apis
+        )
+        self._decision_maker = DecisionMaker(
+            decision_maker_handler=decision_maker_handler
+        )
```

Comment:
@marcofavorito  if no decision maker class is provided we use the default...

Reply:
I think it can be simplified by putting `DefaultDecisionMakerHandler` as a default parameter:

```python
        ...
        is_debug: bool = False,
        max_reactions: int = 20,
        decision_maker_handler_class: Type[DecisionMakerHandler] = DefaultDecisionMakerHandler,
        **kwargs,
    ) -> None:
```